src/components/Scene/CameraLighting.component.tsx

```diff
@@ -1,11 +1,10 @@
 import { useEffect, FC, useRef } from 'react';
 import { useFrame, useThree } from '@react-three/fiber';
-import { Camera, Vector3, DirectionalLight, AmbientLight, SpotLight } from 'three';
+import { Camera, Vector3 } from 'three';
 import { OrbitControls } from 'three-stdlib';
 import { clamp, lerp } from 'src/services';
-import { LightingProps, Required } from 'src/types';
 
-type CameraLightingProps = Required<LightingProps> & {
+type CameraLightingProps = {
```

Comment:
This component is no longer handling lighting effects.
It would be good to rename it so we don't create confusion for the future.
e.g. `CameraControls`